### DISCUSSION TITLE
Combine slow down couple percentage before full

### DISCUSSION THREAD

**Discussion by JustDen1234** [Courseplay/Courseplay_FS22#2526] [Ideas]
It would be nice that the combine slows down before it is completely full. Now it goes on until 100% full, than stops, often keeps driving for a couple of 'meters' because of it's speed. When it slows down (for example to the reverse speed) at say 97 percent, it stops and not skipping crops when continuing.

**Comment by AUZ001** [comment]
+1

**Comment by Tensuko** [comment]
Part of Release 7.2.1.4

**Reply by JustDen1234** [reply]
Thanks !
